posts/2022-01-31-configuring-ocp-for-user-namespace.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configuring ocp for user namespace
+published_date: "2022-01-31 11:45:58 +0000"
+layout: default.liquid
+is_draft: false
+---
+Preparing an openshift cluster for using user namespaces involve
+several steps and hands-over. To simplify the process we are using
+some configurations at freeipa-kustomize that make easier that task.
```

Comment:
suggestion: *freeipa-kustomise* hyperlink to the GitHup repo.

Reply:
thanks! updated